configs/openimages/retinanet_r50_fpn_1x_openiamges.py

```diff
@@ -0,0 +1,17 @@
+_base_ = [
+    '../_base_/models/retinanet_r50_fpn.py',
+    '../_base_/datasets/openimages_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+
+model = dict(bbox_head=dict(num_classes=601))
+
+optimizer = dict(type='SGD', lr=0.08, momentum=0.9, weight_decay=0.0001)
```

Comment:
explain `the gpus * img_per_gpu` in the name of config,  you can refer to the `deformable_detr`